mods/ra/maps/allies-03a/rules.yaml

```diff
@@ -34,6 +34,8 @@ HACKE6:
 		CaptureTypes: building
 	Targetable:
 		RequiresCondition: !jail
+	ExternalConditions:
+		Conditions: jail
```

Comment:
You need to do this for the medics as well here, and for both medics and hacke6 in 3b.